tests/app/celery/test_process_ses_receipts_tasks.py

```diff
@@ -186,20 +189,31 @@ def test_ses_callback_should_retry_if_notification_is_new(mocker):
         assert process_ses_results(ses_notification_callback(reference='ref')) is None
         assert mock_logger.call_count == 0
         assert mock_retry.call_count == 1
+
+
 def test_ses_callback_should_log_if_notification_is_missing(client, _notify_db, mocker):
     mock_retry = mocker.patch('app.celery.process_ses_receipts_tasks.process_ses_results.retry')
     mock_logger = mocker.patch('app.celery.process_ses_receipts_tasks.current_app.logger.warning')
     with freeze_time('2017-11-17T12:34:03.646Z'):
         assert process_ses_results(ses_notification_callback(reference='ref')) is None
         assert mock_retry.call_count == 0
-        mock_logger.assert_called_once_with('notification not found for reference: ref (while attempting update to delivered)')
+        # the multiline indent must be the same as in the application code
+        # for the assertion to completely match
```

Comment:
[question] hmm - does this mean that there's weird extra spaces in that string? I wouldn't have thought that the assert matcher would be able to tell the difference between strings passed as a parameter otherwise.

Reply:
Yeah, there were weird extra spaces. I improved the quoting method and now it's normal again.